dashboard/app/controllers/sessions_controller.rb

```diff
@@ -56,6 +56,27 @@ def reset
     render layout: false
   end
 
+  # GET /lockout
+  # This page is for accounts that are locked until parental permission compliance.
+  def lockout
+    # Basic defaults. If the @pending_email is empty, the request was never sent
+    @pending_email = ""
+    @request_date = DateTime.now
+
+    # Determine the deletion date as the creation time of the account + 7 days
+    @delete_date = current_user.created_at + (3600 * 24 * 7)
```

Comment:
```suggestion
    @delete_date = current_user.created_at + 7.days
```
OR
```suggestion
    @delete_date = current_user.created_at.since(7.days)
```